src/tarea/Tarea3.js

```diff
@@ -24,4 +34,16 @@ export function UncontrolledCheckbox(props) {}
  * debe renderizar tres checkboxes, con nombres "uno", "dos" y "tres", que inicien con valores false, true y false respectivamente.
  */
 
-export function CheckboxList(props) {}
+export function CheckboxList(props) {
+    const names = Object.keys(props.items);
+    const initialValues = Object.values(props.items);
+    return (
+        <div>
+            {names.map((item, i) => (
+                <React.Fragment key={names[i]}>
```

Comment:
Los `React.Fragment` en este caso son innecesarios porque solo tienen un child, pero no afectan negativamente

Reply:
Pensé que era la única forma de asignar keys, los usé porque en la consola me tiraba que tenía que usar keys para identificar los items de una lista. Ahora les saqué el ```React.Fragment``` y le puse un key a ```<UncontrolledCheckbox key={ algo }...```

Reply:
Esto es de acuerdo a algo que estaba leyendo en la documentación, que dice:
_A good rule of thumb is that elements inside the map() call need keys._

https://reactjs.org/docs/lists-and-keys.html#keys

Reply:
sip, lo de las keys está perfecto, pero podés agregarle key a cualquier componente, no es necesario que sea un fragment